mesop/dataclass_utils/dataclass_utils.py

```diff
@@ -8,8 +8,10 @@
 from deepdiff.path import parse_path
 
 from mesop.exceptions import MesopException
+import base64 # Import base64 for encoding
```

Comment:
[nit] Usually don't need to add a comment for why you're importing something. So can remove it.